tests/flakyfunctional/events/01-task.t

```diff
@@ -31,4 +36,4 @@ sed 's/ (after .*)$//' "${SUITE_RUN_DIR}/log/suite/events.log" | sort -u \
 cmp_ok 'actual.events.log' 'expected.events.log'
 #-------------------------------------------------------------------------------
 purge_suite "${SUITE_NAME}"
-exit
+exit
```

Comment:
Vscode has a slightly strange behaviour where it displays trailing newlines as blanklines at the end of the file. Perhaps due to its Microsoft pedigree? This seems pretty stupid to me so there will probably be a configuration or plugin somewhere which can hack this back something a bit more unixey. Otherwise watch out for ⛔ markers at the end of files on GH.

Reply:
UGH